snuba/datasets/outcomes.py

```diff
@@ -75,6 +123,7 @@ def __init__(self) -> None:
                 ("outcome", UInt(8)),
                 ("reason", LowCardinality(Nullable(String()))),
                 ("event_id", Nullable(UUID())),
+                ("size", Nullable(UInt(32))),
```

Comment:
In the original PR this was not nullable, still it is treated as nullable in the processor (since you use a get). You are required to add the column as Nullable in the migration statement (since the table is already there) but this is for the create statement. Is the column supposed to always have a value ?